Tasks/PublishPipelineArtifactV0/task.json

```diff
@@ -9,7 +9,7 @@
     "author": "Microsoft Corporation",
     "version": {
         "Major": 0,
-        "Minor": 151,
+        "Minor": 139,
```

Comment:
The version was 151 before?

Reply:
Yeah, since your changes to the master is not merged yet.